### DISCUSSION TITLE
getting error after moving to nvim 0.10

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#2096] [Q&A]
When opening file /restoring from session

![CleanShot 2023-11-30 at 21 04 27@2x](https://github.com/LazyVim/LazyVim/assets/1625373/2b6dc654-3dd5-4505-936c-d882cd383011)
![CleanShot 2023-11-30 at 21 02 51@2x](https://github.com/LazyVim/LazyVim/assets/1625373/deff45e3-3241-46bf-b83a-3dddc43d6429)

My config:https://github.com/kapral18/config.nvim/blob/ffdb572892a01f74b7328cc8279008136258baf0/init.lua

**Comment by dpetka2001** [comment]
You should update your Neovim version to latest nightly.

**Comment by folke** [comment]
Should be fixed now: https://github.com/LazyVim/LazyVim/commit/8baf9b5459a903c783a1d34ad438e64036a8e15e

Or indeed, update to latest nightly
